Formula/k/kubevpn.rb

```diff
@@ -0,0 +1,25 @@
+class Kubevpn < Formula
+  desc "Offers a Cloud-Native Dev Environment that connects to your K8s cluster network"
+  homepage "https://www.kubevpn.cn"
+  url "https://github.com/kubenetworks/kubevpn.git",
+      tag:      "v2.2.11",
+      revision: "98d88ac542c06e8f66937e7f57b5ceae514b0c30"
+  license "MIT"
+  head "https://github.com/kubenetworks/kubevpn.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["OUTPUT_DIR"] = bin
+    ENV["VERSION"] = version
+    ENV["GIT_COMMIT"] = Utils.git_head
+    ENV["BRANCH"] = Utils.git_branch
+    system "make", "kubevpn"
+  end
+
+  test do
+    system "#{bin}/kubevpn", "version"
```

Comment:
```suggestion
    assert_match version.to_s, shell_output("#{bin}/kubevpn version")
```

One last suggestion from me -- while we're here let's check the version output too.

Reply:
```shell
➜  2.2.11 git:(master) ./bin/kubevpn version
KubeVPN: CLI
    Version: 2.2.11
    DaemonVersion: unknown
    Image: docker.io/naison/kubevpn:2.2.11
    Branch: HEAD
    Git commit: 98d88ac542c06e8f66937e7f57b5ceae514b0c30
    Built time: 2024-06-24 12:32:18
    Built OS/Arch: darwin/arm64
    Built Go version: go1.22.4
➜  2.2.11 git:(master)
```
if check command version output, it show like this. I am confusing how to check substring, but i read cookbook, not found methond/function substring... maybe you can help me, thanks in advance

Reply:
```shell
assert_includes shell_output("#{bin}/kubevpn version"), "Version: #{version}"
```
i found method include ~

Reply:
`assert_match` is used more often in `homebrew/core` and does essentially the same in this case, but yes, `assert_includes` works too.